File: 2023/java/src/test/java/org/frieder/aoc/day9/TestDay9B.java
package org.frieder.aoc.day9;

import org.frieder.aoc.day9.b.Solution9B;
import org.junit.Test;

import java.io.IOException;

import static org.junit.Assert.assertEquals;

public class TestDay9B {
    @Test
    public void testExample() throws IOException {
        assertEquals(2, Solution9B.getResult("../resources/9/test_b.txt"), 0);
    }

    @Test
    public void testResult() throws IOException {
        assertEquals(928, Solution9B.getResult("../resources/9/input.txt"), 0);
    }
}

File: 2023/php/test/TestDay9B.php
<?php declare(strict_types=1);

$loader = require __DIR__ . '/../vendor/autoload.php';

use day9\b\Solution9B;
use PHPUnit\Framework\TestCase;
use utils\TestLogger;

final class TestDay9B extends TestCase
{

    public function testExample(): void
    {
        $this->assertSame(2, Solution9B::getResult("../resources/9/test_b.txt", TestLogger::createTestLogger()));
    }

    public function testResult(): void
    {
        $this->assertSame(928, Solution9B::getResult("../resources/9/input.txt", TestLogger::createTestLogger()));
    }

}